flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/api/PlannerConfigOptions.java

```diff
@@ -30,19 +31,20 @@
 	// ------------------------------------------------------------------------
 	//  Optimizer Options
 	// ------------------------------------------------------------------------
-
+	@Documentation.ExcludeFromDocumentation(value = "Just for corner cases that sql is converted to hundreds of thousands of CNF nodes.")
```

Comment:
Just for corner cases that a simple predicate is converted to too many CNF nodes. ?